### ISSUE TITLE
Emoji keyboard is not user friendly

### ISSUE THREAD

**Issue by kwunyeung** [desmos-labs/mooncake#24]
Is it possible to use the native OS emoji keyboard instead of Flutter package? I am trying to adapt the emoji keyboard but I find it really hard to get a comfortable experience with it.

**Comment by rmontagnin** [comment]
@kwunyeung I've tried searching all the web to find a way to open up the system emoji keyboard, but it looks like Flutter doesn't allow to do it, nor there is any plugin that does this. 

Can you please post a screenshot of the iOS emoji keyboard?

**Comment by kwunyeung** [comment]
![image](https://user-images.githubusercontent.com/932636/84985888-28f7bf80-b170-11ea-998e-6a9a102ffdc7.png)

I see... Discord is also using their own emoji input screen. Let me discuss this with Rooty.

**Comment by rmontagnin** [comment]
@kwunyeung I personally think the best way to handle this particular feature is to continue using the one that Flutter provides. It's going to be a hell lot of work designing a new input system, which IMO is not worth the time/effort right now.

**Comment by kwunyeung** [comment]
@RiccardoM yes I understand. Let’s keep the existing emoji keyboard first. We look for solution later. We may need to build our own interface like Discord some day...
